docs/workflows.rst

```diff
@@ -55,6 +55,19 @@ single reference template (see `Longitudinal processing`_).
         t1w=['sub-01/anat/sub-01_T1w.nii.gz'],
     )
 
+.. important::
+
+    Occasionally, openly shared datasets may contain preprocessed anatomical images
+    as if they are unprocessed.
+    Particularly, brain-extracted images will crash the anatomical workflow of *fMRIPrep*.
+    It is possible to make *fMRIPrep* adapt to such a circumstance by setting the
+    command line argument ``---skull-strip-t1w auto``.
```

Comment:
since this isn't the default, we should document what it does here.

```suggestion
    command line argument ``---skull-strip-t1w auto``, which will use a heuristic to check if each image is already masked
```

Reply:
(this suggestion has been added)